build-system/compile/bento-remap.js

```diff
@@ -66,11 +66,18 @@ const getAllRemappings = once(() => {
   }));
 
   // Allow component cross-dependency
-  const componentRemappings = bentoBundles.map(({name, version}) => ({
-    source: `./src/bento/components/${name}/${version}/${name}`,
-    cdn: `./${name}-${version}.mjs`,
-    npm: `@bentoproject/${getNameWithoutComponentPrefix(name)}`,
-  }));
+  const componentRemappings = bentoBundles.map(({name, version}) => {
+    const nameWithoutPrefix = getNameWithoutComponentPrefix(name);
+    return {
+      source: `./src/bento/components/${name}/${version}/${name}`,
+      cdn: `./${name}-${version}.mjs`,
+      npm:
+        // Special: NPM builds depend on `mustache` directly
+        nameWithoutPrefix === 'mustache'
+          ? 'mustache'
```

Comment:
need to update the `package.json`s for the extensions to add a (peer or regular) dependency on `mustache`.

Reply:
could be neat to support configuring custom npm `dependencies` / `peerDependencies` for each extension in the bundle config as well (not blocking this PR)